apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -996,6 +996,27 @@ fields("log") ->
                     aliases => [file_handlers],
                     importance => ?IMPORTANCE_HIGH
                 }
+            )},
+        {"external",
```

Comment:
There is only ome `opentelemetry` type for now, introducing `external` type appears worthwhile for potential future extensions.  However, it's also possible to simply add `log.opentelemetry` - should be discussed.